### PR TITLE
one of min-available/max-available must be specified when create pdb

### PR DESCRIPTION
**Pull request by wackxu**
**What this PR does / why we need it**:

fix TODO one of min-available/max-available must be specified when create pdb

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/kubectl/pdb_test.go [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+import (
+	apiequality "k8s.io/apimachinery/pkg/api/equality"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/kubernetes/pkg/apis/policy"
+	"testing"
+)
+
+func TestPodDisruptionBudgetV1Generator(t *testing.T) {
```
Comment (foxish) [inline pkg/kubectl/pdb_test.go]: V2Generator?

**Comment by foxish** [comment]
/lgtm
@kubernetes/sig-cli-maintainers for approval.

cc @mengqiy

File: pkg/kubectl/pdb_test.go (hunk 1 above)
Comment (wackxu) [reply pkg/kubectl/pdb_test.go]: @foxish addressed, PTAL

**Comment by foxish** [comment]
/lgtm

Thanks!

**Comment by wackxu** [comment]
@foxish  fix some conflict, PTAL

File: pkg/kubectl/pdb_test.go [hunk 2]
```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```
Comment (zhangxiaoyu-zidif) [inline pkg/kubectl/pdb_test.go]: Copyright 2016 The Kubernetes Authors.
->
Copyright 2017 The Kubernetes Authors.
This is file creation time.

Comment (wackxu) [reply pkg/kubectl/pdb_test.go]: @zhangxiaoyu-zidif addressed

**Pull request merged**